servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/CatalogTests.java

```diff
@@ -2747,12 +2896,95 @@ public void testRegisterExistingTable() {
     Table table = catalog.loadTable(identifier);
     TableOperations ops = ((BaseTable) table).operations();
     String metadataLocation = ops.current().metadataFileLocation();
-    Assertions.assertThatThrownBy(() -> catalog.registerTable(identifier, metadataLocation))
+    assertThatThrownBy(() -> catalog.registerTable(identifier, metadataLocation))
         .isInstanceOf(AlreadyExistsException.class)
         .hasMessageStartingWith("Table already exists: a.t1");
     assertThat(catalog.dropTable(identifier)).isTrue();
   }
 
+  /*
+  @Test
```

Comment:
Why is this test commented out?

Reply:
We don't support "custom metrics reporters" in Nessie. (Same for the new, commented out function above)